CHANGELOG.md

```diff
@@ -4,6 +4,46 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.0-RC2](https://github.com/go-gitea/gitea/releases/tag/v1.11.0-rc2) - 2020-01-18
+* SECURITY
+  * Never allow an empty password to validate (#9682) (#9683)
+  * Prevent redirect to Host (#9678) (#9679)
+* BUGFIXES
+  * Fix markdown anchor links (#9673) (#9840)
```

Comment:
```suggestion
  * Explicitly refer to PR in squash-merge commit message in case of external tracker (#9844) (#9855)
  * Fix markdown anchor links (#9673) (#9840)
```